deps/v8/gypfiles/toolchain.gypi

```diff
@@ -61,7 +61,7 @@
     'v8_android_log_stdout%': 0,
 
     # Force disable libstdc++ debug mode.
-    'disable_glibcxx_debug%': 0,
+    'disable_glibcxx_debug%': 1,
```

Comment:
> Checked STL containers have different name mangling in libstdc++. I think the worst case is that the add-on fails to load at runtime, but not that it silently does the wrong thing.

Some of the functions do get inlined, even in debug builds. (Like, this modification is not just academic; without it, I could observe actual out-of-bound writes.)

> FWIW in V8 5.6 `Release` will return a `std::pair<uint8_t*, size_t>` instead of a vector.

Ok, then at least we don’t need to worry about it. :)